src/tiny_slam/tiny_scan_cost_estimator.h

```diff
@@ -3,11 +3,29 @@
 
 #include "../core/grid_scan_matcher.h"
 
+/*!
+ * \brief Derived class from ScanCostEstimator to calculate "cost" of one scan
```

Comment:
Consider: "An implementation of the scan cost function described in the original tinySLAM paper."